README.md

```diff
@@ -19,13 +19,17 @@ Step 1: install dependencies and run the ground station. In order to run as a us
 ```
 yarn install_dependencies
 yarn csp:clone
-yarn build
-yarn cli <options>
 ```
 
-e.g. `yarn cli -I uart -d /dev/ttyUSB0`
-
-Step 2: before and after development, the existing tests should be run:
+Step 2: Build the ground station. For isolated backend software testing, run:
```

Comment:
why don't we make life a bit easier by bundling all of these setup commands into one `yarn setup` and `yarn setup:gnuradio`

Reply:
you could even add to this setup automated conda and radioconda install checking (and install if not already installed). Ez, low-risk automation to make it easier to deploy on many PCs and/or reinstall!

Reply:
and gpredict

Reply:
I will do this in a future PR - I agree it needs to be done. But we won't be reinstalling this on any other PCs in the near term. I've already done it for the GS and the flatsat, and would prefer to get this PR merged so that we can start using all the important fixes.